allennlp_models/lm/modules/token_embedders/language_model.py

```diff
@@ -76,15 +77,28 @@ def __init__(
 
         # Extract the name of the tokens that the LM was trained on.
         text_field_embedder = dict_config["model"]["text_field_embedder"]
-        token_names = list(text_field_embedder["token_embedders"].keys())
-        if len(token_names) != 1:
-            # We don't currently support embedding with language models trained with multiple
-            # embedded indices.
-            #
-            # Note: We only care about embedded indices. This does not include "tokens" which
-            # is just used to compute the loss in LanguageModel.
-            raise ConfigurationError(f"LM from {archive_file} trained with multiple embedders!")
-        self._token_name = token_names[0]
+        non_empty_embedders = [
+            token_embedder_key
+            for token_embedder_key, token_embedder_value in text_field_embedder[
+                "token_embedders"
+            ].items()
+            if token_embedder_value.get("type") != "empty"
+        ]
+
+        if len(non_empty_embedders) == 0:
+            # Only empty embedders were contained in the language model
+            # We need at least one non-empty embedder in the language model
+            raise ConfigurationError(
+                f"Language model from {archive_file} trained with only empty embedders!"
+            )
+        elif len(non_empty_embedders) > 1:
+            warnings.warn(
+                "Multiple non-empty embedders were found, but we will only use "
+                f"the first one: {non_empty_embedders[0]}",
+                RuntimeWarning,
+            )
```

Comment:
This scenario will not work right. If the LM was trained with multiple embedders, it will need to be run with multiple embedders. It won't work to just leave one out.